static/bidder-info/janet.yaml

```diff
@@ -0,0 +1,17 @@
+maintainer:
+  email: "info@thejmg.com"
+gvlVendorID: 410
```

Comment:
Please confirm your legal team has advised to use Adtelligent's GDPR Vendor ID for the Janet alias.

Reply:
Thanks for notice. JANet confirmed they do not have gvlid